DQM/Integration/python/clients/beampixel_dqm_sourceclient-live_cfg.py

```diff
@@ -5,19 +5,18 @@
 from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
 process = cms.Process("BeamPixel", Run2_2018)
 
-unitTest=False
-if 'unitTest=True' in sys.argv:
-    unitTest=True
+unitTest = False
+if 'unitTest = True' in sys.argv:
```

Comment:
Hi @dinardo, this seems to be a breaking change because you are introducing spaces in a string but the argument is passed in without the spaces: https://github.com/cms-sw/cmssw/blob/master/DQM/Integration/test/runtest.sh#L19

Reply:
Sorry about that.
I fixed it.
Many thanks.
- Mauro.